AutomatedTesting/Gem/PythonTests/Prefab/TestSuite_Periodic.py

```diff
@@ -7,7 +7,7 @@
 
 import pytest
 
-from ly_test_tools.o3de.editor_test import EditorBatchedTest, EditorTestSuite
+from ly_test_tools.o3de.editor_test import EditorBatchedTest, EditorSingleTest, EditorTestSuite
 
```

Comment:
Looks like "EditorBatchedTest" can be removed

Reply:
It'll be re-used as new tests are added. Don't think it hurts to leave it as-is for now.